src/Core/Checkout/Customer/SalesChannel/SwitchDefaultAddressRoute.php

```diff
@@ -30,10 +33,16 @@ class SwitchDefaultAddressRoute extends AbstractSwitchDefaultAddressRoute
      */
     private $customerRepository;
 
-    public function __construct(EntityRepositoryInterface $addressRepository, EntityRepositoryInterface $customerRepository)
+    /**
+     * @var EventDispatcherInterface
+     */
+    private $eventDispatcher;
+
+    public function __construct(EntityRepositoryInterface $addressRepository, EntityRepositoryInterface $customerRepository, EventDispatcherInterface $eventDispatcher)
```

Comment:
you need to change the services.xml and add the new dependency

https://github.com/shopwareBoostDay/platform/blob/master/src/Core/Checkout/DependencyInjection/customer.xml#L254-L257